modules/howtos/examples/KvBasic.kt

```diff
@@ -254,3 +261,44 @@ private suspend fun pessimisticLocking(collection: Collection) {
     )
 // end::pessimisticLocking[]
 }
+
+
+// tag::bulkGet[]
+/**
+ * Gets many documents at the same time.
+ *
+ * @param ids The IDs of the documents to get.
+ * @param maxConcurrency Limits how many operations happen
```

Comment:
David: does this actually work?  FWIW, nn the mobile side, there is a db-wide lock that, absolutely, prevents simultaneous operations.  The only way to do something similar is with a bulk op that takes a block and executes it inside the locks...

Reply:
True, it's not a global limit; it only limits the number of outstanding `get` requests for this one call to `builkGet`.